### DISCUSSION TITLE
remove post message from bar

### DISCUSSION THREAD

**Discussion by the1gofer** [tqdm/tqdm#1187] [Q&A]
Is it possible to remove portions of the post message "[00:20<00:00,  1.27s/it]"?    I don't really care how many itterations per second it's doing.  I see with postfix i can add something .

**Comment by kaisugi** [comment]
You can change your bar style with `bar_format` option!

e.g.,

```python
from tqdm import tqdm
from time import sleep

text = ""
for char in tqdm(["a", "b", "c", "d"], bar_format="{l_bar}{bar}| {n_fmt}/{total_fmt}"):
    sleep(1)
    text = text + char
```

```
100%|██████████| 4/4
```

**Reply by the1gofer** [reply]
Thanks.  Is their anyway to get something like this to work?
`for term in tqdm(Parser.searchTerms, desc="Searching for links on Shepard's website", bar_format=term):`

**Reply by casperdcl** [reply]
er you've got 2 variables called `term` in that line. Should use separate names.

**Comment by the1gofer** [comment]
Right.  I’d like to update the bar with the term that’s being used.

On Tue, Jun 22, 2021 at 12:33 PM Casper da Costa-Luis <
***@***.***> wrote:

> er you've got 2 variables called term in that line
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/tqdm/tqdm/discussions/1187#discussioncomment-906809>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAQKUNCJB6UTMKT3YGMDNV3TUDQOZANCNFSM462FTMUA>
> .
>
-- 
Jason Crews
